continuous_integration/travis/flake8_diff.sh

```diff
@@ -133,6 +133,6 @@ check_files() {
 if [[ "$MODIFIED_FILES" == "no_match" ]]; then
     echo "No file has been modified"
 else
-    check_files "$(echo "$MODIFIED_FILES" )" "--ignore=E501,E731,E12,W503 --exclude=*.sh,*.md,*.yml,*.rst,*.ipynb,*.txt,*.csv,*.vec,Dockerfile*"
+    check_files "$(echo "$MODIFIED_FILES" )" "--ignore=E501,E731,E12,W503 --exclude=*.sh,*.md,*.yml,*.rst,*.ipynb,*.txt,*.csv,*.vec,Dockerfile*,*.c,*.pyx"
```

Comment:
@gojomo We were getting errors like these due to flake8 : 
![flake8_2](https://user-images.githubusercontent.com/10104911/27708414-b873ad2c-5ccd-11e7-8d62-87394f73b737.png)

![flake8_1](https://user-images.githubusercontent.com/10104911/27708415-b8766c10-5ccd-11e7-972b-9ab14b5cf1ab.png)

So although I do agree that there is some style-checking that we might want to do in .pyx files (in the python-like code), to avoid getting errors due to cases similar to the above cases, I thought it would be better to ignore .pyx for flake8 tests.